runtimes/azure-client-runtime/src/main/java/com/microsoft/azure/Graph.java

```diff
@@ -13,6 +13,16 @@
 import java.util.Set;
 
 /**
+ * The edge types in a graph.
+ */
+enum GraphEdgeType {
```

Comment:
yeah that's a requirement for public enums but it's also helpful to understand what each type of edge means here?